packages/modular-scripts/src/start.ts

```diff
@@ -73,11 +65,16 @@ async function start(packageName: string): Promise<void> {
 
   // If you want to use webpack then we'll always use webpack. But if you've indicated
   // you want esbuild - then we'll switch you to the new fancy world.
-  if (!useWebpack || useEsbuild) {
+  if (!useWebpack || useEsbuild || isView) {
```

Comment:
This is untransparent for people that are still using webpack for apps they will probably be expecting webpack for starting a view. I think it would be better to throw an error down the webpack path if it's a view so that people are blocked from that flow, given esbuild is still opt in.

Reply:
Agree. This is going to be temporary until we have Webpack 5, but I will log an error and throw down the webpack path.

Reply:
Done, in `start` and `build` too.

Reply:
There is [another PR](https://github.com/jpmorganchase/modular/pull/1439) that supports Webpack 5 and contains this PR. For the sake of simplicity, I'm keeping them separated.